build_runner/lib/src/build_script_generate/build_script_generate.dart

```diff
@@ -15,30 +15,135 @@ const scriptLocation = '.dart_tool/build/build.dart';
 Future<Null> ensureBuildScript() async {
   var scriptFile = new io.File(scriptLocation);
   // TODO - how can we invalidate this?
-  if (scriptFile.existsSync()) return;
+  //if (scriptFile.existsSync()) return;
   scriptFile.createSync(recursive: true);
   await scriptFile.writeAsString(await _generateBuildScript());
```

Comment:
can we wrap this in a `logTimedAsync` so we can see how long its taking? If its fast I am actually fine w/doing it on startup for every build. Since we are using content hashes now it should still be valid for incremental builds.

Reply:
It's not fast. 2-3 seconds as currently implemented. I'm not sure why it's so slow.

Reply:
I'm going to hold off for another commit for this.

Reply:
sure, that is fine w/me